.github/workflows/build.yml

```diff
@@ -173,3 +173,23 @@ jobs:
         run: cmake --build build/ --config Release
       - name: "test: libterminal"
         run: .\build\src\libunicode\Release\unicode_test.exe
+
+
+  Fedora:
+    name: Fedora
+    runs-on: ubuntu-22.04
+    container: fedora:rawhide
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install build dependencies
+        run: |
+          dnf -y install gcc-c++ cmake ninja-build \
+            fmt-devel range-v3-devel catch-devel unicode-ucd
+      - name: CMake build
+        run: |
+          cmake . -G Ninja -B build -DBUILD_SHARED_LIBS=ON -DLIBUNICODE_UCD_DIR=/usr/share/unicode/ucd
```

Comment:
Why do we set `LIBUNICODE_UCD_DIR` here?

Reply:
The unicode-ucd package in Fedora provides the files that the ucd download from https://www.unicode.org/Public/15.0.0/ucd/UCD.zip.

see: https://github.com/contour-terminal/libunicode/issues/68, https://github.com/contour-terminal/libunicode/pull/69

Reply:
Alright, thanks for pointing this out, forgot about it :(